conda-recipes/pynion-feedstock/.ci_support/linux_noarch.yaml

```diff
@@ -0,0 +1,16 @@
+c_compiler:
+- gcc
+c_compiler_version:
+- '8'
+channel_sources:
+- conda-forge,defaults
```

Comment:
It doesn't appear to need conda-forge as a source, as both packages required are available in main,

Reply:
Agree. I'd seen the same. Decided there was no harm in leaving it just for better commonality between yaml files here and for other noarch packages.